src/vsphere_cpi/spec/integration/nsx_vsphere_spec.rb

```diff
@@ -177,7 +177,7 @@
         cpi_options
       end
 
-      it 'raises an error' do
+      xit 'raises an error' do
```

Comment:
why is this commented?

Reply:
I think you can delete it as this test is no longer valid with your refactoring in cloud.rb